### ISSUE TITLE
After fetching tool, incorrect holder location displayed

### ISSUE THREAD

**Issue by lazerwolf22** [corbinstreehouse/Mach-4-Avid-CNC-Customizations#8]
Greetings!

If set tool 12 to be located at holder 10 and it correctly registers as doing so, however, when I fetch tool 12, the spindle correctly grabs it at holder 10 but then changes the holder location back to the original which was also 12. Example: T12 H10, fetch tool from H10, then shows T12 H12. The same happens with the other tools. When I select "Put Back", the tool is returned to the expected holder regardless of the incorrect holder displayed. So, tool 12 was returned to holder 10 even though T12 H12 was displayed.

Under edit screen, the holder location container is set to "droATCCurrentHeightNum.." I'm guessing that's incorrect?

![Screenshot 2024-09-09 200631](https://github.com/user-attachments/assets/0447b365-8b75-47f9-8caf-257f71cccdef)

**Comment by corbinstreehouse** [comment]
I think it is working right, but let me explain: M6 T5 H5  means "tool change, tool five, activate height for tool 5". The H5 doesn't mean "Holder", but it means "Height". The H[0] indicates what height is active, as a tool can be in the holder but the height doesn't have to be active. So, you'd probably never do " T12 H10" as you'd be selecting tool 12 and activating tool 10's height, which would be weird. There are some instances when you don't want the height active, such as when you are setting the tool height. Does this help clarify it?

**Comment by corbinstreehouse** [comment]
There is no UI indication of what "holder" or "tool fork" or "pocket" is currently active (all mean the same thing)...except when you are on the ATC tools page, the line in green is the pocket active.

**Comment by lazerwolf22** [comment]
Yep, that clarifies it. I've always assumed it was for designating the assigned holder because those numbers always matched so when I went to 11+ tools with only 10 holders I noticed this and it freaked me out when I thought the tool was going to the wrong holder and was going to crash into another tool. Thanks for the clarification on that.

Side note: I thank you nearly every single day when I run my mill and it doesn't move because I didn't turn on the air pressure. Your addition of that with instructions is gold.
